lib/setup-browser.js

```diff
@@ -21,10 +19,16 @@ var mocha = new Mocha({
 if ('{{INVERT}}' === true) { // eslint-disable-line no-constant-condition
   mocha.invert();
 }
-mocha.reporter('{{REPORTER}}', '{{REPORTER_OPTIONS}}');
+
+if (Mocha.reporters['{{REPORTER}}']) {
+  mocha.reporter('{{REPORTER}}', '{{REPORTER_OPTIONS}}');
+} else {
+  mocha.reporter(require('{{REPORTER}}'), '{{REPORTER_OPTIONS}}');
```

Comment:
Mocha is now bundled using Rollup which does not support dynamic requires for CommonJS. This way we pass in the reporter module instead of its name so that Mocha doesn't have to do any module resolution anymore.